Symfony/CS/ConfigInterface.php

```diff
@@ -70,6 +70,20 @@ public function setDir($dir);
     public function getDir();
 
     /**
+     * Sets the directory into which the cache file would be written if caching were enabled.
+     *
+     * @return string
+     */
+    public function setCacheDir($dir);
```

Comment:
@GrahamCampbell 

Also, [this](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1075) clearly shows that users relying on the `ConfigInterface` already can't use the fixer anyway, can they?